configs/server.toml

```diff
@@ -180,6 +180,28 @@ certificate = "certs/iggy.pfx"
 # Password for the TLS certificate, required for accessing the private key.
 password = "iggy123"
 
+# Configuration for the TCP socket
+[tcp.socket]
```

Comment:
Include the `enabled` flag (or some other name), with a default value `false` to check if the custom TCP socket config should be used or not (more info below in the `TcpSocketConfig` section.